app/frontend/wallet/shelley/shelley-trezor-crypto-provider.ts

```diff
@@ -332,7 +335,7 @@ const ShelleyTrezorCryptoProvider = async ({
 
   const prepareByronWitness = (witness: TrezorTypes.CardanoSignedTxWitness): TxByronWitness => {
     const publicKey = Buffer.from(witness.pubKey, 'hex')
-    const chainCode = Buffer.from(witness.chainCode, 'hex')
+    const chainCode = Buffer.from(witness.chainCode || '', 'hex')
```

Comment:
byron witnesses should have a chain code, so this fallback to empty string is really fragile. A runtime asserttion would be more sensible